src/Symfony/Component/Messenger/Bridge/Redis/Tests/Transport/ConnectionTest.php

```diff
@@ -73,6 +73,18 @@ public function testFromDsnWithOptions()
         );
     }
 
+    public function testFromDsnWithTls()
+    {
+        $redis = $this->createMock(\Redis::class);
+        $redis->expects($this->exactly(2))
```

Comment:
I'd rather create 2 separate tests. It would be more readable than mixing the 2 cases on the same mock.

Reply:
That makes sense. 

Thank you. I've updated the PR